scripts/install_nfs.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Check if a dir is exported by the NFS server
```

Comment:
check if there is an exported dir from NFS server... I guess it sounds better..

Reply:
Done.